pyvista/plotting/mapper.py

```diff
@@ -25,17 +25,20 @@ class _BaseMapper(_vtk.vtkAbstractMapper):
 
     _theme = None
```

Comment:
This came from `main` but I'm not sure why there's a class attr here when we're shadowing it in `__init__()` anyway.

Reply:
Need not be a blocker though, feel free to ignore for now. Thanks @akaszynski for the improvements.

Reply:
That's here because we're using `no_new_attr` and without it we'll get:

```
    def __setattr__(self, name, value):
        """Do not allow setting attributes."""
        if hasattr(self, name):
            object.__setattr__(self, name, value)
        else:
>           raise AttributeError(
                f'Attribute "{name}" does not exist and cannot be added to type '
                f'{self.__class__.__name__}'
            )
E           AttributeError: Attribute "_theme" does not exist and cannot be added to 
type DataSetMapper                                                                  

pyvista/utilities/misc.py:176: AttributeError

```